napari/_qt/qt_range_slider.py

```diff
@@ -77,7 +77,13 @@ def __init__(
 
         self.setRange((0, 100) if data_range is None else data_range)
         self.setValues((20, 80) if initial_values is None else initial_values)
-        self.setStep(0.01 if step_size is None else step_size)
+        if step_size is None:
+            # pick an appropriate slider step size based on the data range
+            if data_range is not None:
+                step_size = (data_range[1] - data_range[0]) / 1000
+            else:
+                step_size = 0.001
```

Comment:
I feel like 100 steps was a better default than 1000...

Reply:
Ah, I forgot about the big slider. =D

Well, as I understand it, and correct me if I'm wrong, we're triggering some brief but nonzero computation on each tick of the slider, and currently we don't do this asynchronously, so having too many stops will result in choppiness?

But, you're the one actually playing around with this, so, ignore me! =P

Reply:
let me try to raise it to a ridiculous amount and see if the handle seems to noticeably lag behind the mouse as a I raise it.  If I learn something, i'll update the PR, otherwise, I'll just merge

Reply:
I upped it to one _nonillion_ steps (1e30 😜) and didn't notice any observable difference.  One possibility is that the pixel pitch of the monitor/QCursor position is limiting signals to something quite reasonable?

Reply:
😆 😆 😆 😆  That's awesome!